.circleci/config.yml

```diff
@@ -0,0 +1,57 @@
+version: 2.1
+
+orbs:
+  trellis:
+    executors:
+      python-27:
+        docker:
+          - image: 'circleci/python:2.7-stretch'
+      python-36:
+        docker:
+          - image: 'circleci/python:3.6-stretch'
+
+    jobs:
+      build-python:
+        parameters:
+          executor:
+            type: executor
+        executor: << parameters.executor >>
+        steps:
+          - run: python --version
+          - checkout
+          - restore_cache:
+              keys:
+                - ansible-v1-{{ checksum "requirements.yml" }}
+          - run:
+              name: Install Python dependencies in a venv
+              command: |
+                virtualenv venv
+                . venv/bin/activate
+                pip install ansible
```

Comment:
Yeah as @TangRufus mentioned, this should be locked to a specific version. Let's pick the latest supported version? `2.7.5` right now